tdp/cli/commands/status/generate_stales.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Optional
+
+import click
+
+from tdp.cli.commands.status.utils import (
+    _common_status_options,
+    _print_sch_status_logs,
+)
+from tdp.cli.queries import get_sch_status
+from tdp.cli.session import get_session
+from tdp.cli.utils import check_services_cleanliness
+from tdp.core.cluster_status import ClusterStatus
+from tdp.core.variables import ClusterVariables
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from tdp.core.collections import Collections
+
+
+@click.command()
+@_common_status_options
+def generate_stales(
+    service: Optional[str],
+    component: Optional[str],
+    collections: Collections,
+    database_dsn: str,
+    validate: bool,
+    vars: Path,
+) -> None:
+    """Generate stale components.
+
+    Stales components are components that have been modified and need to be
```

Comment:
For me it is not clear the definition of a modified component. It is necessary to write that a component is modified when its associated variables are modified.

Reply:
I suggest we improve the documentation after having refactored every commands. My argument is that other commands use the notion of stale, some core methods uses this notion too, and I'd rather write something coherent for every methods at once.

Do you agree with that or do you want me to improve this one now?

Reply:
ok